repaygithub/cactus
# CACTUS-311 :: Upgrade @repay/babel-preset and @repay/scripts

The next phase of upgrading the recently release `ui-tools` packages.

### Testing

1. Run `yarn cleanup && yarn build` to make sure everything builds
2. Run `yarn web start` to make sure Storybook starts normally and looks normal.
3. Run `yarn w standard start` to build/run the standard example app.
4. Check out the standard app in Chrome to make sure everything looks good.
5. Check out the standard app in IE.
